pyinfra_cli/main.py

```diff
@@ -96,10 +96,14 @@ def _print_support(ctx, param, value):
     help='Print std[out|err] from operations/facts.',
 )
 @click.option('--user', help='SSH user to connect as.')
+@click.option('--winrm_username', help='WINRM user to connect as.')
```

Comment:
Could we put all of these `--winrm` arguments in a group together, I think that makes more sense.

Also always use dashes here, no underscores, as click will convert them anyway.

Reply:
Ok